src/autoscaler/metricscollector/cmd/metricscollector/main.go

```diff
@@ -208,3 +284,16 @@ func generateGUID(logger lager.Logger) string {
 	}
 	return uuid.String()
 }
+
+func getOwner(logger lager.Logger, conf *config.Config) string {
+	var owner string
+	if strings.TrimSpace(os.Getenv("VCAP_APPLICATION")) != "" {
+		appEnv, _ := cfenv.Current()
+		owner = appEnv.ID
+		logger.Info("ownership found in VCAP_APPLICATION", lager.Data{"owner": owner})
+	} else if conf.DBLock.Owner != "" {
```

Comment:
I wonder that getOwner function is called when conf.EnableDBLock is set to true. 
Am I missing something?